modules/nf-core/muse/call/meta.yml

```diff
@@ -0,0 +1,63 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "muse_call"
+description: pre-filtering and calculating position-specific summary statistics using
+  the Markov substitution model
+keywords:
+  - genomics
+  - wgs
+  - wxs
+  - vcf
+tools:
+  - "MuSE":
+      description: "Somatic point mutation caller based on Markov substitution model
+        for molecular evolution"
+      homepage: "https://bioinformatics.mdanderson.org/public-software/muse/"
+      documentation: "https://github.com/wwylab/MuSE"
+      tool_dev_url: "https://github.com/wwylab/MuSE"
+      doi: "10.1101/gr.278456.123"
+      licence: ["https://github.com/danielfan/MuSE/blob/master/LICENSE"]
+      identifier: ""
+
+input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. `[ id:'sample1' ]`
+    - tumor_bam:
```

Comment:
Does the tool use the indexes of the sorted bam files if they are present? If so, they should be passed in too.

Reply:
i am not sure, from the tool description it seems that it takes care of this inside of the tool (tabix is one of the dependencies)

Reply:
Can you ask the authors please? As often the tools assume that the index is next to the bam file, which will generally work fine with symlinks on a HPC system but isn't true for cloud execution.